caravel/assets/javascripts/reduxUtils.js

```diff
@@ -67,9 +67,13 @@ export function addToArr(state, arrKey, obj) {
 export function enhancer() {
   let enhancerWithPersistState = compose(persistState());
   if (process.env.NODE_ENV === 'dev') {
-    enhancerWithPersistState = compose(
-      persistState(), window.devToolsExtension && window.devToolsExtension()
-    );
+    if (window.devToolsExtension) {
+      enhancerWithPersistState = compose(
+        persistState(), window.devToolsExtension && window.devToolsExtension()
+      );
+    } else {
+      console.log('Please kindly install Redux Devtool Extension to use SQL Lab');
```

Comment:
we could use `console.warn` here to get the warning formatting in the console.
also, we may want to generalize the error message since we will soon be using redux in explore view as well.
`You may encounter errors unless you have Redux Devtool Extension installed -- https://github.com/zalmoxisus/redux-devtools-extension'`

![screenshot 2016-09-30 14 43 52](https://cloud.githubusercontent.com/assets/130878/19007876/5a68693e-871c-11e6-9f72-d25bcc102a5e.png)